dist/index.html

```diff
@@ -38,7 +38,7 @@
       <footer class="container-fluid">
         <hr>
         <div class="pull-right" align="right">For more information visit <a href="https://apigility.org" target="_blank">apigility.org</a></div>
-        <div>&copy; 2013 - 2015 by <a href="http://www.zend.com" target="_blank">Zend Technologies</a> Ltd</div>
+        <div>&copy; 2013 - <script>document.write(new Date().getFullYear());</script> by <a href="http://www.zend.com" target="_blank">Zend Technologies</a> Ltd</div>
```

Comment:
This should actually be in `src/index.html`. I'll make that change on merge; thanks for the patch!